applications/launchpad/gui-react/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { fireEvent, render, screen } from '@testing-library/react'
+import { ThemeProvider } from 'styled-components'
+
+import Checkbox from './'
+
+import themes from '../../styles/themes'
+
+describe('Checkbox', () => {
+  it('should render label', () => {
+    render(
+      <ThemeProvider theme={themes.light}>
+        <Checkbox checked={false} onChange={() => null}>
+          test label
+        </Checkbox>
+      </ThemeProvider>,
+    )
+
+    expect(screen.getByText('test label')).toBeInTheDocument()
+  })
+
+  it('should render svg icon when checked', () => {
+    const { container } = render(
+      <ThemeProvider theme={themes.light}>
+        <Checkbox checked={true} onChange={() => null}>
+          test
+        </Checkbox>
+      </ThemeProvider>,
+    )
+
+    const icon = container.querySelector('svg')
+    expect(icon).toBeInTheDocument()
+  })
+
+  it('should call onChange correctly when label is clicked', () => {
+    const onChange = jest.fn()
+    render(
+      <ThemeProvider theme={themes.light}>
+        <Checkbox checked={false} onChange={onChange}>
+          test
+        </Checkbox>
+      </ThemeProvider>,
+    )
+
+    const label = screen.getByText('test')
```

Comment:
NIT: It's good to define all identifiers at the beginning, ie:

```
const testText = 'test'

render(
      <ThemeProvider theme={themes.light}>
        <Checkbox checked={false} onChange={onChange}>
          {testText}
        </Checkbox>
      </ThemeProvider>,
    )

const label = screen.getByText(testText)
```

Reply:
sure, if you prefer, will include in #175